packages/libs/react-ui/src/components/Breadcrumbs/BreadcrumbsItem.tsx

```diff
@@ -0,0 +1,42 @@
+import { ProductIcon } from '../Icons';
+
+import {
+  iconContainer,
+  itemClass,
+  linkClass,
+  spanClass,
+} from './Breadcrumbs.css';
+
+import React, { FC, ReactNode } from 'react';
+
+export interface IBreadcrumbItemProps {
+  children?: ReactNode;
+  icon?: typeof ProductIcon[keyof typeof ProductIcon];
+  href?: string;
+}
+
+export const BreadcrumbsItem: FC<IBreadcrumbItemProps> = ({
+  children,
+  icon,
+  href,
+}) => {
+  const Icon = icon;
+
+  return (
+    <li className={itemClass}>
+      {Icon && (
+        <div className={iconContainer}>
```

Comment:
shouldn't we use an inline element here instead of a block-level element.
Do we have rules about this stated somewhere?